include/datadog_agent_six.h

```diff
@@ -24,14 +24,19 @@ DATADOG_AGENT_SIX_API six_t *make3();
 DATADOG_AGENT_SIX_API void destroy(six_t *);
 DATADOG_AGENT_SIX_API int init(six_t *, char *);
 DATADOG_AGENT_SIX_API int add_python_path(six_t *, const char *path);
-DATADOG_AGENT_SIX_API six_gilstate_t ensure_gil(six_t *);
 DATADOG_AGENT_SIX_API void clear_error(six_t *);
+DATADOG_AGENT_SIX_API six_gilstate_t ensure_gil(six_t *);
 DATADOG_AGENT_SIX_API void release_gil(six_t *, six_gilstate_t);
-DATADOG_AGENT_SIX_API int get_check(six_t *, const char *name, const char *init_config, const char *instances,
-                                    six_pyobject_t **check, char **version);
+DATADOG_AGENT_SIX_API int get_class(six_t *six, const char *name, six_pyobject_t **py_module,
+                                    six_pyobject_t **py_class);
+DATADOG_AGENT_SIX_API int get_class_version(six_t *, six_pyobject_t *py_class, char **version);
```

Comment:
this is not semantically correct, for wheels the version is at the module level. Let's rename to just `get_version`